example/config/deploy.rb

```diff
@@ -4,3 +4,4 @@
 set :application, 'my_app_name'
 set :repo_url, 'git@github.com:MindscapeHQ/capistrano-raygun-deployment.git'
 set :deploy_to, '/var/www/my_app_name'
+set :raygun_deployment_environments, [:production]
```

Comment:
Yeah could do that. I want to make it explicit that it's not doing anything - the ruby gem gets heaps of support emails about it not delivering errors in dev mode because the default config silently does nothing if it's not running in production mode.